app/javascript/controllers/list_keyboard_navigation_controller.js

```diff
@@ -17,7 +17,10 @@ export default class extends Controller {
 
   getLinkTargetInDirection(direction) {
     const indexOfLastFocus = this.indexOfLastFocus()
-    return this.focusableLinks[indexOfLastFocus + direction]
+    let nextIndex = (indexOfLastFocus + direction) % this.focusableLinks.length
+    if (nextIndex < 0) nextIndex = this.focusableLinks.length - 1
```

Comment:
if last item is focused and you press "down" then next focus will go to the first item

Reply:
Sounds good! Nice improvement.